demos/common/performance_metrics/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+# Copyright (C) 2020 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+find_package(InferenceEngine 2.0 REQUIRED)
+find_package(OpenCV REQUIRED COMPONENTS imgproc)
+
+set(SOURCES metrics.cpp)
+set(HEADERS metrics.h)
+
+# Create named folders for the sources within the .vcproj
+# Empty name lists them directly under the .vcproj
+source_group("src" FILES ${SOURCES})
+source_group("include" FILES ${HEADERS})
+
+add_library(performance_metrics STATIC ${SOURCES} ${HEADERS} ${InferenceEngine_INCLUDE_DIRS})
```

Comment:
Does this need to be a separate library? IMO, it could just be part of `common`. I'm turning `common` into a real library in #1350; once that's merged, the performance metrics could just be added to it.